### PR TITLE
docs(preview-examples): show correct style imports

### PR DESCRIPTION
**Pull request by danielleroux**
## 💡 What is the current behavior?

IX-1605

## 🏁 Checklist

A pull request can only be merged if all of these conditions are met (where applicable):

- [ ] 🦮 Accessibility (a11y) features were implemented
- [ ] 🗺️ Internationalization (i18n) - no hard coded strings
- [ ] 📲 Responsiveness - components handle viewport changes and content overflow gracefully
- [x] 📄 Documentation was reviewed/updated (`pnpm run docs`)
- [ ] 🧪 Unit tests were added/updated and pass (`pnpm test`)
- [ ] 📸 Visual regression tests were added/updated and pass ([Guide](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md#visual-regression-testing))
- [ ] 🧐 Static code analysis passes (`pnpm lint`)
- [x] 🏗️ Successful compilation (`pnpm build`, changes pushed)

### CODE REVIEW

File: packages/documentation/docs/controls/_avatar_code.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ You can also add the avatar to the header, which will turn it into a clickable b
 
 <Playground 
   name="application-header" 
-  examplesByName 
+
```
Comment (matthiashader) [inline packages/documentation/docs/controls/_avatar_code.md:35]: remove empty line

File: packages/documentation/docs/controls/_blind_code.md [hunk 2]
```diff
@@ -4,35 +4,35 @@ import Preview from '@site/src/components/Preview';
...
 <ApiTableSinceTag message="2.0.0" />
 
 <Playground
-  name="blind-variants"
-  height="55rem"
-  examplesByName>
-</Playground>
+name="blind-variants"
+height="55rem"
+
```
Comment (matthiashader) [inline packages/documentation/docs/controls/_blind_code.md:34]: remove empty line

File: packages/documentation/docs/controls/_toast_code.md [hunk 3]
```diff
@@ -15,22 +15,22 @@ import ApiToastConfigReact from './\_toast/react/toast-config.md';
...
-examplesByName></Playground>
+></Playground>
 
 ## Custom toast message
 
 <Playground
 name="toast-custom" height="18rem"
 hideInitalCodePreview
-examplesByName></Playground>
+></Playground>
```
Comment (matthiashader) [inline packages/documentation/docs/controls/_toast_code.md:25]: General: Do we have a format for the attributes? Sometimes we stack them in one line, sometimes every attributes has a single line (+ident)?

File: packages/documentation/docs/controls/application-frame/_application-header_code.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ The application-header can host custom content which will be displayed on the fa
 
 <Playground 
   name="application-header" 
-  examplesByName 
+
```
Comment (matthiashader) [inline packages/documentation/docs/controls/application-frame/_application-header_code.md:15]: remove empty line

File: packages/documentation/docs/controls/application-frame/_application-header_styleguide.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-import Playground from '@site/src/components/PlaygroundV2';
+import Playground from '@site/src/components/PlaygroundV3';
```
Comment (matthiashader) [inline packages/documentation/docs/controls/application-frame/_application-header_styleguide.md:1]: Why do we need the Playground inside the styleguide in general? It seems not to be used

File: packages/documentation/docs/controls/application-frame/_application_code.md [hunk 6]
```diff
@@ -49,7 +49,7 @@ The navigation to another application is implemented via `window.open` (https://
 
 <Playground 
   name="application-app-switch" 
-  examplesByName 
+
```
Comment (matthiashader) [inline packages/documentation/docs/controls/application-frame/_application_code.md:52]: Remove empty lines

File: packages/documentation/docs/controls/application-frame/_content_code.md [hunk 7]
```diff
@@ -13,7 +13,7 @@ The `ix-content` is usually used as layouting component on a single page.
 
 <Playground 
   name="content" 
-  examplesByName 
+
```
Comment (matthiashader) [inline packages/documentation/docs/controls/application-frame/_content_code.md:16]: Remove empty line

File: packages/documentation/docs/controls/application-frame/_content_styleguide.md [hunk 8]
```diff
@@ -1,5 +1,5 @@
 import Tags from './../../auto-generated/ix-content/tags.md';
-import Playground from '@site/src/components/PlaygroundV2';
+import Playground from '@site/src/components/PlaygroundV3';
```
Comment (matthiashader) [inline packages/documentation/docs/controls/application-frame/_content_styleguide.md:2]: Same here: Do we need need the Playground inside the styleguide?

File: packages/documentation/docs/guidelines/_theme-switching_code.md [hunk 9]
```diff
@@ -1,15 +1,15 @@
...
 ## Working with themes during runtime
 
 <ApiTableSinceTag message="1.3.0" />
 
-<PlaygroundV2
+<PlaygroundV3
   name="theme-switcher"
   height="15rem"
-  examplesByName  
+
```
Comment (matthiashader) [inline packages/documentation/docs/guidelines/_theme-switching_code.md:13]: remove empty line

**Comment by matthiashader** [comment]
image
image
Imports getting resolved wrong in stackblitz

File: packages/documentation/src/components/PlaygroundV2/utils.ts [hunk 10]
```diff
@@ -13,6 +13,33 @@ import { themeSwitcher } from '@siemens/ix';
...
+    path = 'angular';
+  }
+
+  if (framework === TargetFramework.REACT) {
+    path = 'react';
+  }
+
+  if (framework === TargetFramework.VUE) {
+    path = 'vue';
+  }
```
Comment (matthiashader) [inline packages/documentation/src/components/PlaygroundV2/utils.ts:31]: Eventually would be more clearly?
  switch (framework) {
    case TargetFramework.ANGULAR:
      path = 'angular';
      break;
    case TargetFramework.REACT:
      path = 'react';
      break;
    case TargetFramework.VUE:
      path = 'vue';
  }

File: packages/documentation/src/components/PlaygroundV3/html-snippets.ts [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { TargetFramework } from '../PlaygroundV2/framework-types';
+import { getBranchPath } from '../PlaygroundV2/utils';
+import { docusaurusFetch } from './fetching';
+
+export async function fetchSourceForHtml(baseUrl: string, name: string) {
+  const snippets: Record<string, string> = {};
+  const tsFile = await docusaurusFetch(`${baseUrl}/${name}.html`, true);
```
Comment (matthiashader) [inline packages/documentation/src/components/PlaygroundV3/html-snippets.ts:15]: if we want to be very precise: const htmlFile

File: packages/documentation/src/components/PlaygroundV3/index.tsx [hunk 12]
```diff
@@ -0,0 +1,352 @@
...
+      }}
+    >
+      There is no example code yet 😱
+    </div>
+  );
+}
+
+function useUrlMapper(activeFramework: TargetFramework) {
+  const baseUrl = useBaseUrl('auto-generated/previews');
+  if (activeFramework === TargetFramework.ANGULAR) {
```
Comment (matthiashader) [inline packages/documentation/src/components/PlaygroundV3/index.tsx:53]: switch (activeFramework) {
  case TargetFramework.ANGULAR:
    return `${baseUrl}/angular`;
  case TargetFramework.REACT:
    return `${baseUrl}/react`;
  case TargetFramework.VUE:
    return `${baseUrl}/vue`;
  case TargetFramework.JAVASCRIPT:
    return `${baseUrl}/web-components`;
  default:
    return baseUrl;  //e.g JS?
}

File: packages/documentation/src/utils/index.ts [hunk 13]
```diff
@@ -0,0 +1,14 @@
...
+ * SPDX-License-Identifier: MIT
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+export function capitalizeFirstLetter(name: string) {
+  if (name.length === 0) {
+    return name;
+  }
+  return name.charAt(0).toUpperCase() + name.slice(1);
```
Comment (matthiashader) [inline packages/documentation/src/utils/index.ts:13]: In order to take care to component that formatted according to kebab case
  return name
    .split('-')
    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
    .join('');

File: packages/documentation/docs/controls/_avatar_code.md (hunk 1 above)
Comment (danielleroux) [reply packages/documentation/docs/controls/_avatar_code.md:35]: It was a prettier auto formatting will be adapted with file change to `.mdx` file. Resolve other too

File: packages/documentation/src/components/PlaygroundV2/utils.ts (hunk 10 above)
Comment (danielleroux) [reply packages/documentation/src/components/PlaygroundV2/utils.ts:31]: No if statements are fine imo

**Review by matthiashader — APPROVED** [review]
lgtm

**Pull request merged**